### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by dminuoso**
Documentation was just missing a word.

### CODE REVIEW

File: src/Control/Comonad.hs [hunk 1]
```diff
@@ -296,7 +296,7 @@ instance ComonadApply Tree where
 -- | A suitable default definition for 'fmap' for a 'Comonad'.
 -- Promotes a function to a comonad.
 --
--- You can only safely use to define 'fmap' if your 'Comonad'
+-- You can only safely use 'liftW' to define 'fmap' if your 'Comonad'
 -- defined 'extend', not just 'duplicate', since defining
```
Comment (RyanGlScott) [inline src/Control/Comonad.hs]: While we're fixing the documentation for this function, I just noticed that this is incorrectly using the past tense here. Can you change `defined` to `defines`?

**Comment by dminuoso** [comment]
Done

File: src/Control/Comonad.hs (hunk 1 above)
Comment (dminuoso) [reply src/Control/Comonad.hs]: Done

**Comment by RyanGlScott** [comment]
Thanks!

**Pull request merged**